<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GalleryWidget</class>
 <widget class="QWidget" name="GalleryWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#widget_gallery_title {
	background-color:#141414;
}
QToolButton {
	background: transparent;
	border: 0px;
	color: #666666;
	font-size: 16px;
	font-weight: 600;
	font-family: PingFang SC, PingFang SC;
}
QToolButton#toolButton_new_image{
	background-color: qlineargradient(x1:0, y1:0, x2:1, y2:0.2, stop:1 #C4FF00, stop:0 #EEFF00);
	border-radius: 15px;
	padding-left: 12px;
	color:black;
}
QListView {
	background-color:#242424;
	border: 0px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_gallery_title" native="true">
     <property name="styleSheet">
      <string notr="true">font-size:12px;</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>48</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>16</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QLabel" name="label_gallery">
        <property name="styleSheet">
         <string notr="true">font-size:20px;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../extres.qrc">:/ResExt/mipmap-mdpi/gallery.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_sel">
        <property name="minimumSize">
         <size>
          <width>64</width>
          <height>39</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>64</width>
          <height>39</height>
         </size>
        </property>
        <property name="text">
         <string>选择</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_share">
        <property name="minimumSize">
         <size>
          <width>64</width>
          <height>39</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>64</width>
          <height>39</height>
         </size>
        </property>
        <property name="text">
         <string>分享</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_copy">
        <property name="minimumSize">
         <size>
          <width>64</width>
          <height>39</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>64</width>
          <height>39</height>
         </size>
        </property>
        <property name="text">
         <string>复制</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_del">
        <property name="minimumSize">
         <size>
          <width>64</width>
          <height>39</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>64</width>
          <height>39</height>
         </size>
        </property>
        <property name="text">
         <string>删除</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_close">
        <property name="text">
         <string>X</string>
        </property>
        <property name="icon">
         <iconset resource="../extres.qrc">
          <normaloff>:/ResExt/mipmap-mdpi/close.png</normaloff>:/ResExt/mipmap-mdpi/close.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_new_image">
        <property name="minimumSize">
         <size>
          <width>72</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>72</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>新建</string>
        </property>
        <property name="icon">
         <iconset resource="../extres.qrc">
          <normaloff>:/ResExt/mipmap-mdpi/new_image.png</normaloff>:/ResExt/mipmap-mdpi/new_image.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>12</width>
          <height>12</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonStyle::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QListView" name="listView">
     <property name="viewMode">
      <enum>QListView::ViewMode::IconMode</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../extres.qrc"/>
 </resources>
 <connections/>
</ui>
